### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
I update the README file:  I add the live demo

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  @TahenyBELGUITH, 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [ ] Kindly fix the file conflicts in the branch.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 > this project is about the last part of the challenge where:
 > -I create a desktop version app
 > -I integrate Figma UI by using Html CSS
-
+live demo: https://tahenybelguith.github.io/Portfolio-mobile-version/
```
Comment (mikenath223) [inline README.md:6]: - [ ] Please fix the conflict in the readme to ensure the branch can be merged.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on the attempt to make the changes requested in the previous review 👏 
There are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._
- [ ] There are still conflicts on the branch. Kindly fix it.

Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
-# Portfolio-mobile-version
+# Portfolio-Desktop-version
 
 > this project is about the last part of the challenge where:
 > -I create a desktop version app
 > -I integrate Figma UI by using Html CSS
 
+[live demo for Portfolio](https://tahenybelguith.github.io/Portfolio-mobile-version)
```
Comment (mikenath223) [inline README.md:7]: - [ ] To fix the conflict perhaps try merging the main branch into this feature branch. Steps:
- On your terminal, run `git pull origin main` to pull any change on the `main` branch.
- Next switch to the `desktop-version` branch.
- Then merge the main branch into the `desktop-version` branch: `git merge origin main`
- Then push the commit created by the merge to the `desktop-version` branch.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Tahtouha, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 💯 👍.
Congratulations! 🎉 🎉 🥂


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have no more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**